examples/tv-casting-app/tv-casting-common/support/CastingStore.cpp

```diff
@@ -539,8 +547,10 @@ CHIP_ERROR CastingStore::WriteAll(std::vector<core::CastingPlayer> castingPlayer
             ReturnErrorOnFailure(tlvWriter.StartContainer(chip::TLV::ContextTag(kCastingPlayerEndpointServerListContainerTag),
                                                           chip::TLV::kTLVType_Structure, serverListContainerType));
             std::vector<chip::ClusterId> serverList = endpoint->GetServerList();
+            ChipLogProgress(AppServer, "CastingStore::WriteAll() writing CastingPlayer Endpoint ServerList:");
```

Comment:
This is not required, you've already added logging at line 553.

Reply:
For line 550, if the server list is empty, line 553 will not be logged. So it’s helpful to see when the server list is empty. Part of our cache write/read issue is that the list is empty, right?
550: ChipLogProgress(AppServer, “CastingStore::WriteAll() writing CastingPlayer Endpoint ServerList:“);
553: ChipLogProgress(AppServer, “CastingStore::WriteAll() clusterId: %d”, clusterId);